go-ducks.js

```diff
@@ -1,4 +1,4 @@
-const consumables = [() => ''];
+const consumables = [() => 'SECRET'];
```

Comment:
Please put this in app.js.

Reply:
got it

Reply:
text fails to render during app execution. Please provide further clarification

Reply:
Please see #15

Reply:
Obviously use your word and not "A"

Reply:
Obviously its all working now ;)